tests/unit/test_data.py

```diff
@@ -75,8 +75,8 @@ def test_exceptions(magnitude, projection):
 
 class TestParticle:
     @staticmethod
-    def test_repr(particle_database):
-        for particle in particle_database:
+    def test_repr(particle_database: ParticleCollection):
+        for particle in particle_database.values():
```

Comment:
Just a comment, the `values()` seems kind of redundant. I guess the `ParticleCollection` interface is `add`, `find` and iteration. Something for another PR though

Reply:
The reason is that the `ParticleCollection` mimics an (immutable) `Mapping`:
https://github.com/ComPWA/expertsystem/blob/26fae1f1b407ad870ec227635e226fa19d1e9a77/expertsystem/data.py#L293